app/views/graders/_ta_grade_distribution_graph.html.erb

```diff
@@ -0,0 +1,35 @@
+<% unless ta.nil? %>
+  <h3><%= ta.first_name + " " + ta.last_name%></h3>
+  <div>
+    (<%= t('browse_submissions.how_many_marked',
+           num_marked: ta.get_num_marked(@assignment),
```

Comment:
You seem to be passing in a local variable `assignment` here, which is good - you should not use the `@assignment` in this case.